CE09OSSM/CE09OSSM_R00009_ingest.csv

```diff
@@ -2,48 +2,48 @@ parser,filename_mask,reference_designator,data_source,status,notes
 # mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00009/cg_data/cpm3/syslog/cpm_status.txt,CE09OSSM-MFC31-00-CPMENG000,recovered_host,Pending,Incorrectly specified dataset and parser
 mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00009/cg_data/dcl36/syslog/*.syslog.log,CE09OSSM-MFD35-00-DCLENG000,recovered_host,Expected,
 mi.dataset.driver.vel3d_cd.dcl.vel3d_cd_dcl_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00009/cg_data/dcl36/vel3d/*.vel3d.log,CE09OSSM-MFD35-01-VEL3DD000,recovered_host,Expected,
-mi.dataset.driver.vel3d_cd.dcl.vel3d_cd_dcl_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00009/instrmts/dcl36/VEL3D_*/*.VEC,CE09OSSM-MFD35-01-VEL3DD000,recovered_inst,Expected,
+mi.dataset.driver.vel3d_cd.dcl.vel3d_cd_dcl_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00009/instrmts/dcl36/VEL3D*/*.VEC,CE09OSSM-MFD35-01-VEL3DD000,recovered_inst,Expected,
```

Comment:
should the ZPLSC lines be commented out like CE07SHSM?

Reply:
No, the VEL3D data from 09 is good with the correct dates. On 07 the instrument clock reset to 2002, so those ingests won't work.